frontend/src/concepts/roleBinding/RoleBindingPermissionsChangeModal.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react'; //import { Modal, ModalVariant } from '@patternfly/react-core/deprecated';
```

Comment:
```suggestion
import * as React from 'react';
```

nitpick but remove the commented out code